examples/pytorch/mnist-ddp/mnist.py

```diff
@@ -0,0 +1,344 @@
+import argparse
+
+from kubeflow.training import Trainer, TrainingClient
+
+
+def train_fashion_mnist(dict):
+
+    import os
+
+    import torch
+    import torch.distributed as dist
+    import torch.nn as nn
+    import torchvision.transforms as transforms
+    from torch.nn.parallel import DistributedDataParallel
+    from torch.optim.lr_scheduler import StepLR
+    from torch.utils.data import DataLoader
+    from torch.utils.data.distributed import DistributedSampler
+    from torchvision.datasets import FashionMNIST
+
+    backend = dict.get("backend")
+    batch_size = dict.get("batch_size")
+    test_batch_size = dict.get("test_batch_size")
+    epochs = dict.get("epochs")
+    lr = dict.get("lr")
+    lr_gamma = dict.get("lr_gamma")
+    lr_period = dict.get("lr_period")
+    seed = dict.get("seed")
+    log_interval = dict.get("log_interval")
+    save_model = dict.get("save_model")
```

Comment:
Can we keep this example as simple as we can and remove all of these args ?
Since we want to use this as a Getting Started example, I would suggest to keep minimal amount of working PyTorch code.
We can create more examples where we showcase the train func args.